src/components/Pagination/Basic/index.tsx

```diff
@@ -1,7 +1,6 @@
 import React from "react";
-import type { StyleProp, ViewStyle } from "react-native";
+import type { StyleProp, ViewStyle, TouchableWithoutFeedback } from "react-native";
 import { View } from "react-native";
-import { TouchableWithoutFeedback } from "react-native-gesture-handler";
```

Comment:
`TouchableWithoutFeedback` should not be imported as `type` only: the component is used later in this file.

I recommend the following: instead:
```
import type { StyleProp, ViewStyle } from "react-native";
import { TouchableWithoutFeedback, View } from "react-native";
```